### DISCUSSION TITLE
[Prettier Plugin] Add twig support

### DISCUSSION THREAD

**Discussion by JeanLucEsser** [tailwindlabs/tailwindcss#8294] [Ideas]
I was expecting to see the plugin work in .twig files as they are mainly just html.
But it looks like this works only on .html files unless I'm missing something.

I think it would make sense to make it work by default on every extension that contains some html such as twig or blade. But maybe that's not as obvious and/or simple as it seems.

Anyway, here's hoping ;)

Congrats on all your work at tailwind labs ;)

**Comment by RichForever** [comment]
Any updates on this?

**Comment by paulhaem** [comment]
Extended support for HTML templating engines like twig and blade would be awesome. Are there any plans for that in the near future?
